src/app/parcels/edit/editParcelForm.tsx

```diff
@@ -109,6 +112,7 @@ const EditParcelForm = ({ parcelId }: EditParcelFormProps): React.ReactElement =
                 prepareParcelDataForForm(parcelData, collectionCentresData.deliveryPrimaryKey)
             );
             setPackingSlotsIsShown(parcelData.packing_slot?.is_shown);
+            setCollectionCentreIsShown(parcelData.collection_centre?.is_shown);
```

Comment:
If we get an undefined collection centre, shall we say the collection centre isn't shown? i.e. use `parcelData.collection_centre?.is_shown === true`

If so, we can use `useState<boolean>` instead of `useState<boolean | undefined>`